src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  transform: translateY(1px);
```

Comment:
seems hardcoded: I guess, if you do not use it, tests fails, right? but it's a proper way to go, try fixing line-height somewhere, or set proper margins/paddings